index.html

```diff
@@ -24,52 +24,91 @@
+        <a href="#products" class="nav-link">Contact</a>
+      </nav>
+ <!--
+
+    Hello Camper!
+
+    For now, the test suite only works in Chrome!
+    Please read the README below in the JS Editor before beginning.
+    Feel free to delete this message once you have read it.
+    Good luck and Happy Coding!
+
+    - The freeCodeCamp Team
   
-  <nav id="nav-bar">
-    <a href="#projects" class="nav-link">Projects</a>
-    <a href="#video" class="nav-link">About </a>
-    <a href="#products" class="nav-link">Contact</a>
-  </nav>
-
-  <section id="welcome-section">
-    <h1> Hey I am Mimic</h1>
-    <p><span id="wedd">a web developer</span></p>
-  </section>
-
-  <section id="projects">
-    <div class="project-tile">
-      <div>
-       <a href="https://codepen.io/freeCodeCamp/full/zNqgVx>" >
-        <img src="https://raw.githubusercontent.com/freeCodeCamp/cdn/master/build/testable-projects-fcc/images/tribute.jpg">
-      </a>
-        <h3> Tribute Page</h3>
-      </div>
-      <div>
-        <img src="https://raw.githubusercontent.com/freeCodeCamp/cdn/master/build/testable-projects-fcc/images/random-quote-machine.png">
-        <h3> Random Queue Machine</h3>
-      </div>
-      <div>
-        <img src="https://raw.githubusercontent.com/freeCodeCamp/cdn/master/build/testable-projects-fcc/images/calc.png">
-        <h3> JavaScript Calculator</h3>
-      </div>
-      
+
+    -->
+      <section id="welcome-section">
+        <h6>Hey I am Mimic</h6>
+        <p><span id="wedd">a web developer</span></p>
+      </section>
+
+      <section id="projects">
+        <div class="project-tile">
+          <div>
+            <a href="https://codepen.io/freeCodeCamp/full/zNqgVx">
+              <img
+                src="https://raw.githubusercontent.com/freeCodeCamp/cdn/master/build/testable-projects-fcc/images/tribute.jpg"
+                alt="freecode"
+              />
+              <!-- - The freeCodeCamp Team -->
+            </a>
+            <h3>Tribute Page</h3>
+          </div>
+          <div>
+            <img
+              src="https://raw.githubusercontent.com/freeCodeCamp/cdn/master/build/testable-projects-fcc/images/random-quote-machine.png"
+              alt="machine"
+            />
+            <h3>Random Queue Machine</h3>
+          </div>
+          <div>
+            <img
+              src="https://raw.githubusercontent.com/freeCodeCamp/cdn/master/build/testable-projects-fcc/images/calc.png"
+              alt="calc"
+            />
+            <h3>JavaScript Calculator</h3>
+          </div>
+        </div>
+      </section>
     </div>
-  </section>
 
-</div>
-   
+ <!--
 
+    Hello Camper!
 
-    <script  src="js/index.js"></script>
+    For now, the test suite only works in Chrome!
+    Please read the README below in the JS Editor before beginning.
+    Feel free to delete this message once you have read it.
+    Good luck and Happy Coding!
+
+    - The freeCodeCamp Team
+  
+
+    -->
```

Comment:
- [ ] Remove comment lines (103-111)

Reply:
- [ ] As mentioned before, you should keep your code clean, please consider deleting comments from lines 99-111.